Common/pom.xml

```diff
@@ -13,25 +13,30 @@
 
     <dependencies>
         <dependency>
+            <groupId>nl.lumc.sasc</groupId>
+            <artifactId>BiopetUtils</artifactId>
+            <version>summary-0.9.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
             <version>6.8</version>
         </dependency>
         <dependency>
             <groupId>org.scalatest</groupId>
-            <artifactId>scalatest_2.11</artifactId>
+            <artifactId>scalatest_2.10</artifactId>
```

Comment:
yes, this is because I reuse some code in biopet for the summary in utils. Because biopet itself is in scala 2.10 and the testing was in 2.11 I needed to downgrade. I hope I'm able to upgrade biopet/queue to scala 2.11 or 2.12 in future.